.github/workflows/call-build-linux-packages.yaml

```diff
@@ -72,6 +72,72 @@ jobs:
           echo ::set-output name=BUILD_TYPE::$BUILD_TYPE
         shell: bash
 
+  call-build-capture-source:
+    # Capture source tarball and generate checksum for it
+    name: Extract any supporting metadata
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    needs:
+      - call-build-legacy-check
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref }}
+          path: source
+
+      - name: Checkout legacy packaging code
+        if: needs.call-build-legacy-check.outputs.build-type == '1.8'
+        uses: actions/checkout@v3
+        with:
+          repository: fluent/fluent-bit-packaging
+          path: packaging
+
+      - name: Create tarball and checksums
+        run: |
+          tar -czvf $SOURCE_FILENAME_PREFIX.tar.gz -C source --exclude-vcs .
```

Comment:
can't this be a script?

Reply:
my thinking was it relies on the legacy checkout above (although with no more 1.8 releases maybe ok now)